frontend/src/components/playlist.js

```diff
@@ -1,19 +1,254 @@
 import React, { Component } from 'react';
-import logo from '../logo.svg';
+import { Input, Button, Select } from 'antd';
+import { Container, Row, Col } from 'reactstrap';
+import 'antd/dist/antd.css';
+import '../styles/playlist.css'
```

Comment:
I didnt see this sooner with the Navbar stylesheet but lets keep the style sheets with the components. So the project structure will look like components/Playlist/ Inside of the playlist folder is playlist.js and playlist.css